project_index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+require_once('functions.php');
+$db = new PDO('mysql:host=db; dbname=collection_project', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would suggest moving the database connection into it's own function - it will make it easier to get a new connection on a different page (like your add page)

Reply:
Note that because this function will touch the database, it cannot be unit tested